BLSQ/iaso
# SLEEP-1499 Support embedding of Superset dashboard in Iaso

This PR contains a v1 for embedding Superset dashboards in Iaso, using the `Pages` model.

Specifically:
- Add a new pages template that supports embedding a new type of page "SUPERSET"
- Add an API endpoint POST `/api/superset/token` that creates a Superset "guest token" for a specific dashboard

Related JIRA tickets : https://bluesquare.atlassian.net/browse/SLEEP-1499

**Note that this is a v1. In order to make this properly usable for regular users, I think we should have a place where you can configure the connection to such a dashboard for a given account.**

![image](https://github.com/user-attachments/assets/7d6abb1c-a0f7-4b5d-9a08-8f21efe0676e)

## How to test

1. Create a Superset dashboard, make it embeddable, this will give you a `dashboard_id`.
2. Create an embedded page of type "Superset". Set the `dashboard_id` of the dashboard you want to embed.
3. Visit the page, you should now see your dashboard.
4. In the admin page you can configure the UI config options for your dashboard.
example config:
```json
{
  "hideTab": true,
  "hideTitle": true,
  "hideChartControls": true,
  "filters": {
    "visible": false,
    "expanded": false
  },
}
```
(On Trypelim, I have only the `hideTab` and `hideTitle` set. Filters are useful and chart controls allow things like exporting CSV data)
